src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -395,6 +398,26 @@ private Integer getWeekOfDate(ZonedDateTime date) {
         return localDate.get(weekOfYear);
     }
 
+    /**
+     * Fetches Course Management Detail View data from repository and returns a DTO
+     *
+     * @param courseId id of the course
+     * @param exerciseIds the ids of the exercises of the course
+     * @return The DTO for the course management detail view
+     */
+    public CourseManagementDetailViewDTO getStatsForDetailView(Long courseId, Set<Long> exerciseIds) {
+        var dto = new CourseManagementDetailViewDTO();
+        var course = this.courseRepository.findByIdElseThrow(courseId);
+        dto.setCourse(course);
+
+        dto.setNumberOfStudentsInCourse(Math.toIntExact(userRepository.countUserInGroup(course.getStudentGroupName())));
+        dto.setNumberOfTeachingAssistantsInCourse(Math.toIntExact(userRepository.countUserInGroup(course.getTeachingAssistantGroupName())));
+        dto.setNumberOfInstructorsInCourse(Math.toIntExact(userRepository.countUserInGroup(course.getInstructorGroupName())));
```

Comment:
You're missing the editor that got introduced yesterday.

Reply:
Thanks, forget to add the manual fetching, added in 36c3339 :)